We do not support stevedore.sphinxext on Solaris.

Not intended for upstream

--- stevedore-5.6.0/stevedore/tests/test_sphinxext.py
+++ stevedore-5.6.0/stevedore/tests/test_sphinxext.py
@@ -12,10 +12,13 @@
 
 """Tests for the sphinx extension"""
 
+import pytest
+pytestmark = pytest.mark.skip("We do not support sphinxext on Solaris")
+
 import importlib.metadata
 
 from stevedore import extension
-from stevedore import sphinxext
+#from stevedore import sphinxext
 from stevedore.tests import utils
 
 
